crates/uniffi-bindgen-react-native/src/bindings/react_native/gen_typescript/oracle.rs

```diff
@@ -147,7 +148,7 @@ impl CodeOracle {
             FfiType::Callback(name) => self.ffi_callback_name(name),
             FfiType::Struct(name) => self.ffi_struct_name(name),
             FfiType::Reference(inner) => self.ffi_type_label_by_reference(inner),
-            FfiType::VoidPointer => "biginh".to_string(),
+            FfiType::VoidPointer => "/*pointer*/ bigint".to_string(),
```

Comment:
placeholder?  I think long-term wrapping pointers in a class for TS/JS might be the right call.

Reply:
Yes, I have that in my head as a task called: **Make the JS garbage collector to tell Rust when an object goes out of scope**.

Note to self: start filing GH issues.